roles/common/templates/logrotate_rules.j2

```diff
@@ -0,0 +1,120 @@
+# {{ ansible_managed | comment }}
+# Logrotate configuration for {{ item.name }}
+
+{% for path in item.paths %}
+{{ path }}
+{% endfor %} {
+{% if item.options.daily is defined and item.options.daily %}
+    daily
+{% elif item.options.weekly is defined and item.options.weekly %}
+    weekly
+{% elif item.options.monthly is defined and item.options.monthly %}
+    monthly
+{% elif item.options.yearly is defined and item.options.yearly %}
+    yearly
```

Comment:
I think we want to standardize much more than this. This approach allows too much variability - I don't want us to have any logs that rotate monthly or yearly.